Components/Metrics/PCAMetric/itkPCAMetric_F_multithreaded.hxx

```diff
@@ -882,7 +882,7 @@ PCAMetric<TFixedImage, TMovingImage>::LaunchGetSamplesThreaderCallback() const
 {
   /** Setup local threader. */
   // \todo: is a global threader better performance-wise? check
-  auto local_threader = ThreaderType::New();
+  auto local_threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might improve the performance of PCAMetric ("PCAMetric_F_multithreaded") 👍